docs/src/pinn/poisson_annulus.md

```diff
@@ -0,0 +1,228 @@
+# Poisson Equation
+
+In this example, we solve the Poisson equation on an annulus to demostrate how NeuralPDE can solve differential equations in complex geometries.
```

Comment:
using coordinate maps, etc. This should get a bit more explanation at the beginning, with a nice source that goes into more depth, etc.